Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM python:3 as base
+
+#Docker always runs a root 
+# SO by default /root/.poetry/bin 
+# From get-poetry.py
+# HOME = expanduser("~")
+# POETRY_HOME = os.environ.get("POETRY_HOME") or os.path.join(HOME, ".poetry")
+# POETRY_BIN = os.path.join(POETRY_HOME, "bin")
+# POETRY_ENV = os.path.join(POETRY_HOME, "env")
+# POETRY_LIB = os.path.join(POETRY_HOME, "lib")
+# POETRY_LIB_BACKUP = os.path.join(POETRY_HOME, "lib-backup")
+
+ENV POETRY_HOME=/usr/poetry 
+ENV PATH=$PATH:$POETRY_HOME/bin
+
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python 
+
+WORKDIR /my-work-dir
+
+COPY todo_app ./todo_app
+COPY pyproject.toml poetry.toml poetry.lock .env ./
```

Comment:
You can't `COPY` `.env` because it's in your `.dockerignore` file, any you shouldn't need to anyway since you're running the containers with `--env-file .env`

Reply:
I spotted that after the commit thanks removed